Dockerfile.server

```
@@ -12,16 +12,19 @@ RUN yarn install --frozen-lockfile
 
 COPY . .
 
-RUN yarn lerna bootstrap
+RUN yarn --network-timeout 100000 lerna bootstrap 
 RUN rm -rf /app/packages/server/dist/ && yarn build --scope=server
-
+RUN rm -rf /app/packages/client/dist/ && rm -rf /app/packages/client/dist-ssr/ && yarn build:ssr && yarn build:client
```

```suggestion
RUN yarn build:ssr && yarn build:client
```
Эти строчки лишние. 

Нужно создать в корне файл `.dockerignore` с содержимым:
node_modules
packages/*/dist
packages/*/dist-ssr
packages/*/node_modules

---

Ок